app/javascript/pages/country/widget/widgets/widget-tree-gain/widget-tree-gain-component.js

```diff
@@ -2,24 +2,51 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 
 import WidgetDynamicSentence from 'pages/country/widget/components/widget-dynamic-sentence';
+import WidgetNumberedList from 'pages/country/widget/components/widget-numbered-list';
+import NoContent from 'components/no-content';
+import COLORS from 'pages/country/data/colors.json';
 
 import './widget-tree-gain-styles.scss';
 
 class WidgetTreeCoverGain extends PureComponent {
   render() {
-    const { getSentence } = this.props;
+    const { locationNames, loading, data, settings, getSentence } = this.props;
```

Comment:
We no longer need locationNames and loading. These are held in the parent 'Widget' component.